src/components/ranking-chart/ranking-chart-component.jsx

```diff
@@ -76,20 +21,18 @@ const RankingChart = ({
   selectedFilterOption,
   handleSearchChange,
   handleCountryClick,
-  scrollPosition,
+  scrollIndex,
   searchTerm,
 }) => {
   const [hasScrolled, changeHasScrolled] = useState(false);
+
   const tableRef = useRef();
   useEffect(() => {
-    if (scrollPosition > -1 && tableRef.current) {
-      const foundElement = document.getElementById(`country-${scrollPosition}`)
-      if (foundElement) {
-        foundElement.scrollIntoView();
-        tableRef.current.scrollBy(0, -10);
-      }
+    const ROW_HEIGHT = 24;
+    if (scrollIndex > -1 && tableRef.current) {
+      tableRef.current.scroll(0, ROW_HEIGHT * scrollIndex);
```

Comment:
I'm still seeing the same scroll displacement (do you experience the same)? Maybe is related with my browser version?

Reply:
I can' reproduce this problem anymore neither in Chrome or firefox 😕  Are you talking about that white space due to the parent div scrolling?